src/components/DocumentationTopic.vue

```diff
@@ -30,6 +30,14 @@
           <slot name="above-hero-content" />
         </template>
         <slot name="above-title" />
+        <Hierarchy
```

Comment:
I think we should conditionally remove this for the "minimal" view of a page that shows up in the quick navigation preview.

Right now, it looks broken since it always matches the title of the page and doesn't include the full hierarchy—but instead of fixing that, I think it makes sense to just remove it in the more compact view anyway.

(Something like `v-if="!enableMinimized"` should probably do the trick)

Reply:
Another scenario where we may want to remove this is when there are no parent items to click on—this is happening for top-level framework pages right now, and it looks odd since it just copies the title of the page (now that the documentation link is gone).

Reply:
> Another scenario where we may want to remove this is when there are no parent items to click on—this is happening for top-level framework pages right now, and it looks odd since it just copies the title of the page (now that the documentation link is gone).

I agree. This is tracked in a different radar: rdar://121060726 (Remove breadcrumbs from top level framework pages)

Reply:
> I think we should conditionally remove this for the "minimal" view of a page that shows up in the quick navigation preview.

https://github.com/apple/swift-docc-render/pull/772/commits/27b98f3d807a047095c534759b6503271cd691de

Reply:
>I think we should also remove this for the IDE mode.

https://github.com/apple/swift-docc-render/pull/772/commits/4d05719930d885d1a533646c2a0564bdf0de5385